web/src/components/Button.tsx

```diff
@@ -160,3 +185,215 @@ function getSize(size: SizeOptions, type: string, isIconOnly: boolean) {
     }
   }
 }
+
+export function FAQButton({
```

Comment:
Not a big fan of putting all of these in the Button.tsx file :)
To me they are not a "basic" component, and thus should live elsewhere. 
Perhaps these (and potentially more) could live under a new feature-folder called something like `/features/about`? The idea being to group components that are part of the same "feature", despite being used in different places (panel and modal).

Reply:
I agree in principle but nothing really felt right so I put them in button, at least for now.

I'll happily move them once everything is ready to be reviewed 👍🏼

Currently I'm holding off on doing the last style changes until #6855 is merged.

Reply:
@madsnedergaard how would you feel about putting it in a buttons sub folder in components and splitting it out in individual files?

I think that might be the most extensible option?

Reply:
I personally don't feel like they are "core components" used to build up features, but we're also getting into nitty-gritty details so it's not something we need to argue too much about 😄